benchmarks/mmmu/mmmu_utils.py

```diff
@@ -0,0 +1,85 @@
+import ast
```

Comment:
We can just call this "utils.py" since it's already in  the `mmmu/` directory

Reply:
Thanks, made this change.